app/index.js

```diff
@@ -129,6 +129,7 @@ module.exports = generators.Base.extend({
       this.prompt(prompts, function(answers) {
         if (answers.dir !== '.') {
           this.destinationRoot(answers.dir);
+          this.appdir = answers.dir;
```

Comment:
@rob-deans : as mentioned when we discussed this, I don't think this is needed any more.